python/cudf/cudf/core/column/lists.py

```diff
@@ -48,6 +50,24 @@ def offsets(self):
     def list(self, parent=None):
         return ListMethods(self, parent=parent)
 
+    def to_arrow(self):
+        offsets = self.offsets.to_arrow()
+        elements = self.elements.to_arrow()
+        pa_type = pa.list_(elements.type)
+
+        if len(elements) == elements.null_count:
+            elements = pa.nulls(len(elements))
+            pa_type = pa.list_(elements.type)
```

Comment:
Still need to handle this to only do `self.elements.to_arrow()` if `len(elements) != elements.null_count`

Reply:
https://github.com/rapidsai/cudf/pull/6318#discussion_r500559336